src/EnergyPlus/HWBaseboardRadiator.cc

```diff
@@ -1592,20 +1588,19 @@ namespace HWBaseboardRadiator {
         int WaterInletNode;
         int WaterOutletNode;
         auto &Iter = state.dataHWBaseboardRad->Iter;
```

Comment:
`Iter` is int. Should use value instead of ref?

Reply:
It depends on where and how this local is used. There is really no benefit to using a referenced scalar other than readability, and even then it's not a good pattern. This reference is used once at line 1599. I would/will delete this line and use `state.dataHWBaseboardRad->Iter` at line 1599.

Reply:
You're both correct. `Iter` just didn't come up in my `grep` for partial names in this pass (I used `Time`, `Flag`, and `Err` until the total count exceeded 100). Is `Iter` used at all for anything? Can it just be deleted?

Reply:
Within this function, `Iter` seems to be used only at line 1599.
Actually, I'm not sure what this field is used for. Other than lines 1590 and 1599, no other places seem to refer to this field.